### PR TITLE
Fr 1596 general email

### PR DESCRIPTION
**Pull request by ocarleton**
Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/FR-1596

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: charts/finrem-cos/values.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ java:
     IDAM_API_URL: "https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net"
     IDAM_S2S_URL : "http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
     DOCUMENT_GENERATOR_SERVICE_API_BASEURL : "http://finrem-dgcs-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
-    FINREM_NOTIFICATION_SERVICE_BASE_URL : "http://finrem-ns-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+    FINREM_NOTIFICATION_SERVICE_BASE_URL : "http://finrem-ns-pr-147.service.core-compute-preview.internal"
```
Comment (ocarleton) [inline charts/finrem-cos/values.yaml:11]: to be backed out before merge/after testing

File: charts/finrem-cos/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 name: finrem-cos
 apiVersion: v1
 home: https://github.com/hmcts/finrem-case-orchestration-service
-version: 0.0.8
+version: 0.0.9
```
Comment (johnoneill46) [inline charts/finrem-cos/Chart.yaml:4]: No need to bump Chart version as no chart files will be changed

File: audit.json [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  "10023_Information Disclosure - Debug Error Messages_http://finrem-cos-aat.service.core-compute-aat.internal/case-orchestration/update-case_POST": "ignore",
+  "10023_Information Disclosure - Debug Error Messages_http://finrem-cos-aat.service.core-compute-aat.internal/case-orchestration/general-letter-start_POST": "ignore",
+  "10023_Information Disclosure - Debug Error Messages_http://finrem-cos-aat.service.core-compute-aat.internal/case-orchestration/consented/set-defaults_POST": "ignore",
+  "10023_Information Disclosure - Debug Error Messages_http://finrem-cos-aat.service.core-compute-aat.internal/case-orchestration/update-latest-consent-order_POST": "ignore",
+  "10023_Information Disclosure - Debug Error Messages_http://finrem-cos-aat.service.core-compute-aat.internal/case-orchestration/remove-details_POST": "ignore",
+  "100000_A Client Error response code was returned by the server_http://finrem-cos-aat.service.core-compute-aat.internal/forms/form-type/validate-ocr_POST": "ignore",
+  "100000_A Client Error response code was returned by the server_http://finrem-cos-aat.service.core-compute-aat.internal/transform-exception-record_POST": "ignore"
+}
```
Comment (ocarleton) [inline audit.json:61]: This is bought in by this pr being in sync with master and develop being a bit behind. its already in master and nothing to do with this pr

File: security.sh [hunk 4]
```diff
@@ -1,7 +1,9 @@
...
+export LANG=C.UTF-8
+zap-api-scan.py -t ${TEST_URL}/v2/api-docs -f openapi -S -d -u ${SecurityRules} -P 1001
 cat zap.out
-zap-cli --zap-url http://0.0.0.0 -p 1001 report -o /zap/api-report.html -f html
-cp /zap/api-report.html functional-output/
-zap-cli -p 1001 alerts -l Informational
\ No newline at end of file
+curl --fail http://0.0.0.0:1001/OTHER/core/other/jsonreport/?formMethod=GET --output report.json
+cp *.* functional-output/
+zap-cli --zap-url http://0.0.0.0 -p 1001 alerts -l Medium --exit-code False
```
Comment (ocarleton) [inline security.sh:9]: Same as above, bought in from master, not changed in this PR. Caused by develop being a bit behind

File: charts/finrem-cos/Chart.yaml (hunk 2 above)
Comment (ocarleton) [reply charts/finrem-cos/Chart.yaml]: reverted 👍

**Pull request merged**